buildSrc/src/main/groovy/bio.terra.java-common-conventions.gradle

```diff
@@ -38,6 +38,12 @@ repositories {
     }
 }
 
+// spring-boot-gradle-plugin 3.2.5 pulls in opentelemetry-bom 1.31.0.
+// It must have version >= 1.34.1 for compatibility with terra-common-lib 1.1.11:
+ext['opentelemetry.version'] = '1.36.0'
+// Spring Boot 3.2.4 pulls in io.netty:netty-bom 4.1.107.Final which is impacted by CVE-2024-29025.
+ext['netty.version'] = '4.1.108.Final'
```

Comment:
🎉 this pin can be removed: you've pulled in Spring Boot 3.2.5, which moves past this vulnerable version to 4.1.109.Final.

https://github.com/spring-projects/spring-boot/blob/v3.2.5/spring-boot-project/spring-boot-dependencies/build.gradle#L1084